src/main.js

```diff
@@ -88,6 +91,16 @@ try {
     } else if (hasWarn) {
       result = 'warning';
     }
+
+    if (!hasFailure && output_json) {
+      core.setOutput(
+        'json-output',
+        JSON.stringify(
+          YamlLibrary.docs.length == 1 ? YamlLibrary.docs[0] : { documents: YamlLibrary.docs }
```

Comment:
I'm guessing there is a reason to differentiate between one document and multiple? For my own curiosity do you mind explaining? I'd normally think it would be easier to have an object with a `documents` property that is treated as an array that can be enumerated over.

Reply:
After reviewing https://github.com/im-open/import-stewardship-sam-json/pull/2 I'm guessing it's differentiated because that action can take in either. I think it's a good idea for the `import-stewardship-sam-json` action to handle both cases, but it seems like we could simplify logic here and always have the object with the documents property and not worry about the length.

Reply:
This would probably make easier to use later on.  I'll have it always return something like this:

```json
{
  doucments: [<documents1>, <document2>, ...]
}
```